rpcs3/Emu/Io/Dimensions.cpp

```diff
@@ -411,17 +411,20 @@ bool dimensions_toypad::remove_figure(u8 pad, u8 index, bool save, bool lock)
 
 	// When a figure is removed from the toypad, respond to the game with the pad they were removed from, their index,
 	// the direction (0x01 in byte 6 for removed) and their UID
-	std::array<u8, 32> figure_change_response = {0x56, 0x0b, pad, 0x00, figure.index, 0x01};
-	std::memcpy(&figure_change_response[6], figure.data.data(), 7);
-	if (save)
+	if (full_remove)
 	{
+		std::array<u8, 32> figure_change_response = {0x56, 0x0b, pad, 0x00, figure.index, 0x01,
+			figure.data[0], figure.data[1], figure.data[2],
+			figure.data[4], figure.data[5], figure.data[6], figure.data[7]};
```

Comment:
same typo ?
```suggestion
			figure.data[3], figure.data[4], figure.data[5], figure.data[6]};
```

Reply:
This is meant to be 0, 1, 2, 4, 5, 6, 7, originally I thought it was just the first 7 bytes but real figures have their UID set in 0-2, 4-7 rather than 0-6

Reply:
Well, it would be nice if these things were documented somewhere in a comment.